Automattic/newspack-plugin
# fix(ras): js module imports

### All Submissions:






### Changes proposed in this Pull Request:



Fixes a compile error when using the `watch` command.

### How to test the changes in this Pull Request:

1. While on `feat/add-standalone-newsletter-modal`, run `npm run watch` to start a watch session.
2. Visit the site front-end and observe JS errors in the console which prevent the RAS JS from executing:

![image](https://github.com/Automattic/newspack-plugin/assets/2230142/c8a5d5ca-b7ac-48aa-8f85-e7188709f38d)

3. Check out this branch, restart the watch command, and confirm that the errors no longer occur and that the RAS JS features work as expected.

### Other information:
